internal/oscap/oscap.go

```diff
@@ -70,3 +76,15 @@ func IsProfileAllowed(profile string, allowlist []Profile) bool {
 
 	return false
 }
+
+func GetAutotailorFile(profile string) (string, string, *fsnode.Directory, error) {
+	newProfile := fmt.Sprintf("%s_osbuild_tailoring", profile)
+	filepath := fmt.Sprintf("%s/%s", autotailorDirPath, "tailoring.xml")
```

Comment:
Not very important, but `filepath.Join()` would be nicer here for avoiding potential double `//` if the `autotailorDirPath` happens to have a trailing `/`.